pontoon/base/models.py

```diff
@@ -3252,8 +3282,10 @@ class TranslationMemoryEntry(models.Model):
     translation = models.ForeignKey(
         Translation, null=True, on_delete=models.SET_NULL, related_name="memory_entries"
     )
-    locale = models.ForeignKey(Locale)
-    project = models.ForeignKey(Project, null=True, related_name="memory_entries")
+    locale = models.ForeignKey(Locale, models.CASCADE)
+    project = models.ForeignKey(
+        Project, models.CASCADE, null=True, related_name="memory_entries"
+    )
```

Comment:
I suggest we `SET_NULL` if `project` is deleted, same as we do for `entity` and `translation`. The TM entry would still be useful.